Docs.jsx

```diff
@@ -1,8 +1,21 @@
-export default function Docs(){
-    return (
-        <div>
-          I am an "expensive" component... which really just means that I was
-          code-split 😉
-        </div>
-    )
-}
\ No newline at end of file
+import './stateSignal';
+import { Router,Outlet, Link} from '@tanstack/react-location';
+import { routes,location } from './Router';
+import { Suspense } from 'react';
+import Header from './components/Header/Header';
+import Sidepanel from './Sidepanel';
+
+
+function Docs() {
+  return (
+    <Suspense fallback={<div>loading</div>}>
+      <Sidepanel>
+        <Router rouutes={routes} location={location}>
```

Comment:
is it 'rouutes'? Maybe it is a typo? Can you check?

Reply:
Yes, this was a typo